grpc-server/workflow_test.go

```diff
@@ -58,8 +58,8 @@ func TestCreateWorkflow(t *testing.T) {
 		"FailedCreatingWorkflow": {
 			args: args{
 				db: mock.DB{
-					GetTemplateFunc: func(ctx context.Context, fields map[string]string) (string, string, error) {
-						return "", templateData, nil
+					GetTemplateFunc: func(ctx context.Context, fields map[string]string) (string, string, string, error) {
```

Comment:
Instead of returning an empty string, maybe we can return a pre-defined name for mock (e.g., `MockedTemplate`?).